comment/action.yml

```diff
@@ -18,48 +18,27 @@ inputs:
     description: 'Default to {{ github.token }}. This is the default GitHub token available to actions and is used to post comments. The default token permissions (https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions) work fine; `pull-requests: write` is required if you need to customize these.'
     default: '${{ github.token }}'
     required: false
-outputs:
-  body:
-    description: 'The body of comment that was posted.'
-    value: ${{ steps.compost-comment.outputs.body }}
 runs:
   using: "composite"
   steps:
-    - name: Run infracost output
+    - name: Post infracost comment to a github pull-request
```

Comment:
```suggestion
    - name: Post Infracost comment to a GitHub pull request
```

Reply:
fixed in https://github.com/infracost/actions/pull/57/commits/afaa32f1d57d7816f04968f24bbacfe0df99cbaa